RxSwift/Disposables/DisposeBag.swift

```diff
@@ -93,4 +93,14 @@ extension DisposeBag {
             _disposables += disposables
         }
     }
+
+    /// Convenience function allows an array of disposables to be gathered for disposal.
+    public func insert(_ disposables: [Disposable]) {
```

Comment:
DRY - No reason to "rewrite" this method twice. You can use the splatted/variadic splatted version, make it an array and pass it down to this method IMO

Reply:
Didn't occur to me that the veriadic array was an array I could pass into the second version. Thanks.